__tests__/app.test.js

```diff
@@ -145,4 +145,90 @@ describe('6', () => {
+        .then(({body}) => {
+            body.articles.forEach(article => {
+                expect(article).toEqual(
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
+                        body: expect.any(String),
+                        topic: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(Number)
+                    })
+                )
+            })
+        })
+    });
+    test('should return 200 and articles sorted by date and topic', () => {
+        return request(app)
+        .get("/api/articles/?topic=mitch")
+        .expect(200)
+        .then(({body}) => {
+            body.articles.forEach(article => {
+                expect(article).toEqual(
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
+                        body: expect.any(String),
+                        topic: "mitch",
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(Number)
+                    })
+                )
+            })
+        })
+    });
+    test('should return 404 and articles not found with invalid topic', () => {
+        return request(app)
+        .get("/api/articles/?topic=banana")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.message).toBe("no articles found")
+        })
+    });
+})
+
+describe('11. GET /api/articles (queries)', () => {
+    test('returns 200 with valid inputs', () => {
+        return request(app)
+        .get("/api/articles/?sort_by=title&order=ASC")
+        .expect(200)
+        .then(({body}) => {
+            body.articles.forEach(article => {
+                expect(article).toEqual(
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
+                        body: expect.any(String),
+                        topic: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(Number)
+                    })
+                )
+            })
+
+        })  
+    });    
+    test('returns 400 with invalid inputs', () => {
+        return request(app)
+        .get("/api/articles/?sort_by=banana&order=ASC")
+        .expect(400)
+        .then(({body}) => {
+            expect(body.message).toBe("invalid input")
+        })  
+    });    
```

Comment:
You are not checking the sort order in your test assertions. I'd suggest using `jest-sorted` to check this. You should separate out the testing for these two queries so that you are testing:

 - [ ] Status 200, accepts `sort_by` query, e.g. `?sort_by=votes`
  - [ ] Status 200, accepts `order` query, e.g. `?order=desc`
  - [ ] Status 400. invalid `sort_by` query, e.g. `?sort_by=bananas`
  - [ ] Status 400. invalid `order` query, e.g. `?order=bananas`